IBX-4853: Displaying language name instead of its code in language switcher (#2139)
ibexa/documentation-developer: docs/templating/twig_function_reference/storefront_twig_functions.md

<p>Active currency code: {{ currency.code }}</p>
```

## `ibexa_storefront_get_language_name_by_code()`

> `ibexa_storefront_get_language_name_by_code()` returns language name instead of its code in language switcher.

Reviewer: It can be used anywhere:
```suggestion
`ibexa_storefront_get_language_name_by_code()` displays language name based on its code or locale.
```
